fairlearn/metrics/_group_metric_set.py

```diff
@@ -32,15 +27,12 @@
 _PREDICTION_BINARY_CLASSIFICATION = 'binaryClassification'
 _MODEL_NAMES = 'modelNames'
 _SCHEMA = 'schemaType'
-_GROUP_METRIC_SET = 'groupMetricSet'
+_DASHBOARD_DICTIONARY = 'dashboardDictionary'
 _VERSION = 'schemaVersion'
 
-_UNSUPPORTED_MODEL_TYPE = "The specified model_type of '{0}' is not supported"
-_DICT_TOO_MANY_Y_PRED = 'Too many y_pred values in dictionary'
-
 BINARY_CLASSIFICATION = 'binary_classification'
 REGRESSION = 'regression'
-_allowed_model_types = frozenset([BINARY_CLASSIFICATION, REGRESSION])
+_allowed_prediction_types = frozenset([BINARY_CLASSIFICATION, REGRESSION])
 
```

Comment:
We should also have `PROBABILISTIC_BINARY_CLASSIFICATION`.

Reply:
Yes, but if we're keeping this private pending a review of the API and generated JSON, adding that now is just more stuff which will potentially need a substantial update later.